src/Consolonia.Themes/Fluent/FluentTheme.axaml

```diff
@@ -8,15 +8,15 @@
                 <ResourceInclude Source="/Fluent/FluentColors.axaml" />
             </ResourceDictionary.MergedDictionaries>
 
-            <x:String x:Key="ThemeLeftArrowSymbol">˂</x:String>
-            <x:String x:Key="ThemeRightArrowSymbol">˃</x:String>
-            <x:String x:Key="ThemeUpArrowSymbol">˄</x:String>
-            <x:String x:Key="ThemeDownArrowSymbol">˅</x:String>
+            <x:String x:Key="ThemeLeftArrowSymbol">&lt;</x:String>
```

Comment:
maybe keep the unicode chars for fluent and use these for Turbo?

Reply:
They just looked but, I thought this is what we discussed here:
![image](https://github.com/user-attachments/assets/110a17c8-8832-4f95-88e4-a7acee06576c)
Currently in fluent theme it looks like this:
![image](https://github.com/user-attachments/assets/0608e6af-2778-4597-855a-119f04a98f1c)
![image](https://github.com/user-attachments/assets/70f49146-fb86-4311-b00b-9a9c12dc5ce6)
![image](https://github.com/user-attachments/assets/7cf8d307-328c-4f8d-bfad-af9caaf9b406)
Or you think it's better?

Reply:
I've moved this to discuss in a separate PR https://github.com/jinek/Consolonia/pull/324